front/app/modules/commercial/content_builder/admin/components/ContentBuilderTopBar/index.tsx

```diff
@@ -18,25 +20,50 @@ import {
   Title,
 } from '@citizenlab/cl2-component-library';
 
-// Utils
+// utils
 import { isNilOrError } from 'utils/helperUtils';
 
 // i18n
 import messages from './messages';
 import { FormattedMessage } from 'utils/cl-intl';
-import { withRouter } from 'react-router';
 
-// libraries
+// routing
 import clHistory from 'utils/cl-router/history';
+import { withRouter } from 'react-router';
+
+// services
+import {
+  addContentBuilderLayout,
+  PROJECT_DESCRIPTION_CODE,
+} from '../../../services/contentBuilder';
 
 const ContentBuilderPage = ({ params: { projectId } }) => {
+  const [loading, setLoading] = useState(false);
+
+  const { query } = useEditor();
   const localize = useLocalize();
+  const locale = useLocale();
   const project = useProject({ projectId });
 
   const goBack = () => {
     clHistory.goBack();
   };
 
+  const handleSave = async () => {
+    if (!isNilOrError(locale)) {
+      try {
+        setLoading(true);
+        await addContentBuilderLayout(
+          { projectId, code: PROJECT_DESCRIPTION_CODE },
+          { craftjs_jsonmultiloc: { [locale]: JSON.parse(query.serialize()) } }
+        );
+      } catch {
+        // Do nothing
```

Comment:
Should we add any additional error handling here if the add doesn't work?

Reply:
For now - we are not handling this error, we can ask design for a mechanism to do so in a future iteration - definitely a good idea to do so 👍